.github/workflows/pest-tests.yml

```
@@ -0,0 +1,91 @@
+        key: composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('composer.lock') }}
+        restore-keys: composer-${{ runner.os }}-${{ matrix.php-version }}-
+
+    - name: Cache Node Dependencies
+      uses: actions/cache@v4
+      with:
+        path: node_modules
+        key: node-${{ runner.os }}-${{ hashFiles('package-lock.json') }}
+        restore-keys: node-${{ runner.os }}-
+
+    - name: Setup Environment
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)